### DISCUSSION TITLE
I wonder if anyone has installed it on Mac M1 without issue

### DISCUSSION THREAD

**Discussion by tbaloch** [madMAx43v3r/chia-plotter#959] [General]
I wonder if anyone has installed it on Mac M1 without issues.

**Comment by indyramp** [comment]
For what it's worth, I have, two months ago. Alas, I only had the Mac Mini M1 with 8GB RAM and 256GB SSD (it was all they had in stock at my local Apple Store locations when I wanted to buy it in November. 

My times were suboptimal, as were my notes at the time... using an external Thunderbolt 3 NVMe drive. About 312 minutes for mostly default settings as I recall. I'm seeing 182 minutes with -r 4, 159 minutes at -r 8. 

I have a 16GB MBA now, so when I find some time I'll give that one a shot. I suspect the new higher-memory MBP models announced this past week will do much better, but they're not in the budget just yet.

**Comment by brianwfreeman** [comment]
Today was the first time ever I was able to install it on an M1 without issue. I am creating my first plot now on a 13" MBP with an M1. So far it is 2x as fast as my iMac with Intel i7 (and that had a RAMdisk!)
